about.html

```diff
@@ -499,10 +400,10 @@ <h4 class="title">
                 <div class="col-lg-3">
                     <div class="items">
                         <div class="title">
-                            Iteck - 1st eCommerce Dashboard
+                            AICTA - 1st Tax Agent
                         </div>
                         <small class="text">
-                            Over 25 years working in IT services developing software applications and mobile apps for
+                            Over 25 years working in tax services developing x  and x for
```

Comment:
This project is a new program for Mario. We'll have to come back to this at a later date when we get more details.

Commenting to leave a marker.